Remove unneeded pack_to_tuple_t helper

ktf opened the pull request:
Remove unneeded pack_to_tuple_t helper

Drop the pack_to_tuple_t helper which is not particularly needed
and adds an implicit dependency of "Framework/Pack.h" on <tuple>.

alibuild commented:
Error while checking build/O2/fullCI for f1f32c25d3e36d41f5da7ba05078edbc44284390 at 2024-06-19 18:05:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/slc8_x86-64/GCC-Toolchain/v12.2.0-alice1-15/include/c++/12.2.0/bits/utility.h:75:61: error: incomplete type 'std::tuple_size<o2::framework::pack<o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>::RowViewBase<o2::soa::DefaultIndexPolicy, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>::RowViewBase<o2::soa::DefaultIndexPolicy, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > > >' used in nested name specifier
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/test/benchmark_EventMixing.cxx:184:45: error: no match for 'operator*' (operand type is 'o2::framework::GroupedCombinationsGenerator<int, o2::soa::CombinationsBlockStrictlyUpperSameIndexPolicy<o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, int, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > >::iterator' {aka 'o2::framework::GroupedCombinationsGenerator<int, o2::soa::CombinationsBlockStrictlyUpperSameIndexPolicy<o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, int, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > >::GroupedIterator'})
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/test/benchmark_EventMixing.cxx:185:42: error: cannot convert '<brace-enclosed initializer list>' to 'const std::tuple<float, float>&'
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/include/Framework/GroupedCombinations.h:178:9: error: using invalid field 'o2::framework::GroupedCombinationsGenerator<T1, GroupingPolicy, BP, G, As>::GroupedIterator::mCurrentGrouped'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13234/f1f32c25d3e36d41f5da7ba05078edbc44284390/build_O2_fullCI/pretty.html).

ktf committed:
Remove unneeded pack_to_tuple_t helper

Drop the pack_to_tuple_t helper which is not particularly needed
and adds an implicit dependency of "Framework/Pack.h" on <tuple>.

alibuild commented:
Error while checking build/O2/fullCI for b967fe9e37934a849b73292f857bb7a1c966e266 at 2024-10-28 18:45:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/slc8_x86-64/GCC-Toolchain/v13.2.0-alice1-6/include/c++/13.2.0/bits/utility.h:75:61: error: incomplete type 'std::tuple_size<o2::framework::pack<o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>::RowViewBase<o2::soa::DefaultIndexPolicy, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>::RowViewBase<o2::soa::DefaultIndexPolicy, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > > >' used in nested name specifier
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/test/benchmark_EventMixing.cxx:184:45: error: no match for 'operator*' (operand type is 'o2::framework::GroupedCombinationsGenerator<int, o2::soa::CombinationsBlockStrictlyUpperSameIndexPolicy<o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, int, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > >::iterator' {aka 'o2::framework::GroupedCombinationsGenerator<int, o2::soa::CombinationsBlockStrictlyUpperSameIndexPolicy<o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, int, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes> >, o2::framework::ColumnBinningPolicy<o2::aod::collision::PosX, o2::aod::collision::PosY>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::collision::BCId, o2::aod::collision::PosX, o2::aod::collision::PosY, o2::aod::collision::PosZ, o2::aod::collision::CovXX, o2::aod::collision::CovXY, o2::aod::collision::CovYY, o2::aod::collision::CovXZ, o2::aod::collision::CovYZ, o2::aod::collision::CovZZ, o2::aod::collision::Flags, o2::aod::collision::Chi2, o2::aod::collision::NumContrib, o2::aod::collision::CollisionTime, o2::aod::collision::CollisionTimeRes>, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> >, o2::soa::Table<o2::framework::OriginEnc{4476737}, o2::soa::Index<0, -1>, o2::aod::track::CollisionId, o2::aod::track::TrackType, o2::aod::track::X, o2::aod::track::Alpha, o2::aod::track::Y, o2::aod::track::Z, o2::aod::track::Snp, o2::aod::track::Tgl, o2::aod::track::Signed1Pt, o2::aod::track::IsWithinBeamPipe<o2::aod::track::X>, o2::aod::track::Px<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Py<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha>, o2::aod::track::Pz<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::PVector<o2::aod::track::Signed1Pt, o2::aod::track::Snp, o2::aod::track::Alpha, o2::aod::track::Tgl>, o2::aod::track::Energy<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Rapidity<o2::aod::track::Signed1Pt, o2::aod::track::Tgl>, o2::aod::track::Sign<o2::aod::track::Signed1Pt>, o2::soa::Marker<1> > >::GroupedIterator'})
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/test/benchmark_EventMixing.cxx:185:42: error: cannot convert '<brace-enclosed initializer list>' to 'const std::tuple<float, float>&'
/sw/SOURCES/O2/13234-slc8_x86-64/0/Framework/Core/include/Framework/GroupedCombinations.h:178:9: error: using invalid field 'o2::framework::GroupedCombinationsGenerator<T1, GroupingPolicy, BP, G, As>::GroupedIterator::mCurrentGrouped'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13234/b967fe9e37934a849b73292f857bb7a1c966e266/build_O2_fullCI/pretty.html).
